src/DBFTPlugin/Consensus/ConsensusService.cs

```diff
@@ -145,18 +145,21 @@ private void OnStart()
             started = true;
             if (!dbftSettings.IgnoreRecoveryLogs && context.Load())
             {
-                // For Now, only checking Transactions for Priority 
+                // For Now, only checking Transactions for Priority
```

Comment:
remove this comment because you improved the mechanism

Reply:
Sure.